.github/workflows/eamxx-v1-testing.yml

```diff
@@ -53,9 +54,22 @@ jobs:
           pattern=$(IFS=\|; echo "${paths[*]}")
 
           # Use the GitHub API to get the list of changed files
-          response=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-            "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.number }}/files")
-          changed_files=$(echo "$response" | grep -o '"filename": *"[^"]*"' | sed 's/"filename": *//; s/"//g')
+          # There are page size limits, so do it in chunks
+          page=1
+          while true; do
+            response=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+              "https://api.github.com/repos/E3SM-Project/scream/pulls/${{ github.event.number }}/files?per_page=100&page=$page")
```

Comment:
This should not hard code "scream" as the repo name. That said, we decided we're going to roll back to paths filter, and get rid of the pre-process-pr job, so I'm ok leaving this as is, and create a PR right after.

Reply:
This is the reason why the eamxx-v1 and eamxx-sa workflows did not run on this PR.